### PR TITLE
Add or update NPCs from PFS 1-02

### PR DESCRIPTION
**Pull request by TMunsie**

### CODE REVIEW

File: packs/data/pfs-season-1-bestiary.db/mitflit-pfs-1-02.json [hunk 1]
```diff
@@ -0,0 +1,1438 @@
...
+                    "value": "offensive"
+                },
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p>The mitflit sneak deals an extra [[/r {1d6}[precision]]]{1d6 precision damage} to @Compendium[pf2e.conditionitems.Flat-Footed]{Flat-Footed} creatures.</p>"
```
Comment (avagdumortesin) [inline packs/data/pfs-season-1-bestiary.db/mitflit-pfs-1-02.json:1161]: PFS Sneak Attack; no other diffs

File: packs/data/pfs-season-1-bestiary.db/gwibble-3-4.json [hunk 2]
```diff
@@ -1099,13 +1191,18 @@
                     "value": ""
                 }
             },
+            "flags": {
+                "core": {
+                    "sourceId": "Item.7DCGpPolOAXznr5M"
+                }
+            },
```
Comment (SpartanCPA) [inline packs/data/pfs-season-1-bestiary.db/gwibble-3-4.json:1198]:
```suggestion
```

File: packs/data/pfs-season-1-bestiary.db/gwibble-3-4.json [hunk 3]
```diff
@@ -1214,18 +1321,20 @@
...
-                },
-                "weapon": {
-                    "value": ""
+                }
+            },
+            "flags": {
+                "core": {
+                    "sourceId": "Item.E3PTkrzQuAZbvYbB"
                 }
             },
```
Comment (SpartanCPA) [inline packs/data/pfs-season-1-bestiary.db/gwibble-3-4.json:1330]:
```suggestion
```

File: packs/data/pfs-season-1-bestiary.db/gwibble-3-4.json [hunk 4]
```diff
@@ -1302,9 +1487,14 @@
                     "value": ""
                 }
             },
+            "flags": {
+                "core": {
+                    "sourceId": "Item.ZZILWT48DShnJ7wT"
+                }
+            },
```
Comment (SpartanCPA) [inline packs/data/pfs-season-1-bestiary.db/gwibble-3-4.json:1494]:
```suggestion
```

File: packs/data/pfs-season-1-bestiary.db/mitflit-sneak.json [hunk 5]
```diff
@@ -1057,16 +1191,21 @@
                     "value": ""
                 }
             },
+            "flags": {
+                "core": {
+                    "sourceId": "Item.7DCGpPolOAXznr5M"
+                }
+            },
```
Comment (SpartanCPA) [inline packs/data/pfs-season-1-bestiary.db/mitflit-sneak.json:1198]:
```suggestion
```

File: packs/data/pfs-season-1-bestiary.db/mitflit-sneak.json [hunk 6]
```diff
@@ -1097,13 +1236,121 @@
                     "value": ""
                 }
             },
+            "flags": {
+                "core": {
+                    "sourceId": "Item.xLvRc1w2ZmVmbScI"
+                }
+            },
```
Comment (SpartanCPA) [inline packs/data/pfs-season-1-bestiary.db/mitflit-sneak.json:1243]:
```suggestion
```

File: packs/data/pfs-season-1-bestiary.db/mitflit-sneak.json [hunk 7]
```diff
@@ -1140,9 +1408,14 @@
                     "value": ""
                 }
             },
+            "flags": {
+                "core": {
+                    "sourceId": "Item.ZZILWT48DShnJ7wT"
+                }
+            },
```
Comment (SpartanCPA) [inline packs/data/pfs-season-1-bestiary.db/mitflit-sneak.json:1415]:
```suggestion
```

**Review by SpartanCPA — CHANGES_REQUESTED** [review]

**Review by SpartanCPA — APPROVED** [review]

**Pull request merged**